src/openforms/appointments/tests/test_endpoints.py

```diff
@@ -42,7 +39,7 @@ def test_good_token_and_submission_redirect_and_add_submission_to_session(self):
             response = self.client.get(endpoint)
 
         expected_redirect_url = (
-            furl("http://maykinmedia.nl/afspraak-annuleren")
+            furl("http://maykinmedia.nl/myform#afspraak-annuleren")
```

Comment:
```suggestion
            furl("http://maykinmedia.nl/myform/afspraak-annuleren")
```
we don't do client-side routing based on hash-fragments, but actual URLs :)